src/main/webapp/release.html

```diff
@@ -9,63 +9,117 @@
   </head>
 
   <body>
-     <div class="jumbotron">
-        <h1>SDK Release</h1>      
+    <div class="jumbotron">
+        <h1>SDK Releases</h1>  
     </div>  
 
     <div class="row">
-        <div class="col col-md-border" >
-            <div class="card text-white">
-                <div class="card">
-                    <h2 class="card-title">M78</h2>
+
+        <div class="col" >
+            <div class="row justify-content-center">
+                <div class="card text-white">
+                    <div class="card">
+                        <div class="card-header">M78</div>
+                          <div class="card-body">
+                              <p>Enrolled SDKs: 85</p>
+                              <p>Cold Freeze: August 18, 2020</p>
+                              <p>Launch Deadline: August 21, 2020</p>
```

Comment:
-> Launchcal Deadline. Please apply same change to the other lines in this file.